app/build.gradle.kts

```diff
@@ -186,3 +186,46 @@ dependencies {
 
     lintChecks(libs.slack.lint.compose)
 }
+
+dependencies {
+    koverReport {
+        defaults {
+            mergeWith("debug")
+            html {
+                onCheck = false
+                setReportDir(layout.buildDirectory.dir("artifacts/reports/kover/coverageResults"))
+            }
+        }
+    }
+
+    kover(projects.ivyBase)
```

Comment:
Can we automatically add all subprojects here?
Maybe using `projects.forEach` or something similar